File: app/src/main/java/org/take365/Engine/Network/Models/AuthorModel.java
package org.take365.Engine.Network.Models;

import java.io.Serializable;

/**
 * Created by evgeniy on 08.02.16.
 */
public class AuthorModel implements Serializable {
    public int id;
    public String url;
    public String username;
    public StoryImageThumbModel userpic;
    public StoryImageThumbModel userpicLarge;
}

File: app/src/main/java/org/take365/Network/Models/AuthorModel.kt
package org.take365.Network.Models

import java.io.Serializable

/**
 * Created by evgeniy on 08.02.16.
 */
data class AuthorModel(
        var id: Int,
        var url: String?,
        var username: String?,
        var userpic: StoryImageThumbModel?,
        var userpicLarge: StoryImageThumbModel?
) : Serializable
